test/integration-tests/IntegrationTests.Helpers/GlobalSuppressions.cs

```diff
@@ -1,7 +1,18 @@
-// This file is used by Code Analysis to maintain SuppressMessage
```

Comment:
Perhaps preserver this comment below the header.

Reply:
Comment restored.